sitemap.xml

```diff
@@ -1,23 +1,21 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8"?>
 <urlset
     xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd"
 >
-    <url>
-        <loc>https://deranged.dk/</loc>
-    </url>
-    <url>
-        <loc>https://deranged.dk/the-team/</loc>
-    </url>
-    <url>
-        <loc>https://deranged.dk/cases/</loc>
-    </url>
-    <url>
-        <loc>https://deranged.dk/privacy/</loc>
-    </url>
-    <url>
-        <loc>https://deranged.dk/explainers/</loc>
-    </url>
-    <url>
-        <loc>https://deranged.dk/explainers/small-deliveries</loc>
-    </url>
+    {% for page in site.pages %}
+        {% if page.sitemap-include == "yes" %}
+        <url>
+            <loc>https://deranged.dk{{ page.url | remove: "index.html" }}</loc>
```

Comment:
Just in case we ever create a URL where `index.html` is significant (and not the last part of a path) I think it would be slightly more safe to use [remove_last](https://shopify.dev/docs/api/liquid/filters/remove_last)

Imagine e.g. we do a blog post with slug `/blog/the-history-of-index.html`. This code would turn it into `https://deranged.dk/blog/the-history-of-`.

Reply:
Actually, that case would also break with my suggestion...

If we *really* wanted to be precise and avoid future confusion, we should replace via a regex, and replace `/index.html$` (that is slash, followed by "index.html", at the very end of the string) with `/`.

Reply:
Fixed